fvm/crypto/crypto_test.go

```diff
@@ -132,6 +132,21 @@ func TestVerifySignatureFromRuntime(t *testing.T) {
 	})
 
 	t.Run("BLS verification tag size > 32 bytes should pass", func(t *testing.T) {
+		ok, err := crypto.VerifySignatureFromRuntime(
+			crypto.NewDefaultSignatureVerifier(),
+			[]byte("signature"),
+			string("a very large tag with more than thirty two bytes"),
+			[]byte("some random message"),
+			[]byte("public kye"),
+			runtime.SignatureAlgorithmECDSA_P256,
+			runtime.HashAlgorithmKECCAK_256,
+		)
+		require.Error(t, err)
+		require.Equal(t, err.Error(), "[Error Code: 1051] invalid value (a very large tag with more than thirty two bytes): tag length (48) is larger than max length allowed (32 bytes).")
+		require.False(t, ok)
+	})
+
+	t.Run("ECDSA verification tag size > 32 bytes shouldn't pass", func(t *testing.T) {
```

Comment:
I think there is a typo mix up in the test title, the title here got swapped with the one in line 134

Reply:
yup, fixed it